packages/@react-native-windows/telemetry/src/telemetry.ts

```diff
@@ -162,6 +159,8 @@ export class Telemetry {
     }
 
     Telemetry.client.config.disableAppInsights = Telemetry.isTest;
+    Telemetry.client.config.disableStatsbeat = true;
+    Telemetry.client.getStatsbeat().enable(false);
```

Comment:
curious why two props are needed

Reply:
unfortunately there's a bug that, by that point, the statsbeat is already off and running in the background. https://github.com/microsoft/ApplicationInsights-node.js/issues/943

Reply:
my hope is they fix their bug and this works and I can remove the environment variable.